### PR TITLE
Server coin

### PR DESCRIPTION
**Pull request by Rigidity**

### CODE REVIEW

File: src/server_coin.rs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    ff15ff17ffff0181ff8080ffff01ff0bff05ff0bff1780ffff01ff088080ff01
+    80ff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff0effff04ff02ff
+    ff04ff09ff80808080ffff02ff0effff04ff02ffff04ff0dff8080808080ffff
+    01ff0bffff0101ff058080ff0180ff018080
+    "
+);
+
+pub const MIRROR_PUZZLE_HASH: TreeHash = TreeHash::new(hex!(
+    "
+    b10ce2d0b18dcf8c21ddfaf55d9b9f0adcbf1e0beb55b1a8b9cad9bbff4e5f22
```
Comment (Yakuhito) [inline src/server_coin.rs:72]: I assume this is correct

**Review by Yakuhito — CHANGES_REQUESTED** [review]
Good stuff! Please also make a note of the breaking changes like [here](https://github.com/Datalayer-Storage/DataLayer-Driver/pull/3) so MT has an easier time migrating.

Also, update README.md to include the new functions  :)

File: src/server_coin.rs (hunk 1 above)
Comment (Rigidity) [reply src/server_coin.rs:72]: Yes, added a test

Comment (Yakuhito) [reply src/server_coin.rs:72]: many tanks

**Comment by Rigidity** [comment]
> Good stuff! Please also make a note of the breaking changes like [here](https://github.com/Datalayer-Storage/DataLayer-Driver/pull/3) so MT has an easier time migrating.
> 
> Also, update README.md to include the new functions :)

There shouldn't be any breaking changes

**Review by Yakuhito — APPROVED** [review]

**Pull request merged**